### PR TITLE
chore(account): refactor data ressource workflow

### PR DESCRIPTION
**Pull request by remyleone**

### CODE REVIEW

File: scaleway/data_source_account_ssh_key.go [hunk 1]
```diff
@@ -11,35 +11,26 @@ import (
...
 
 func dataSourceScalewayAccountSSHKey() *schema.Resource {
+	dsSchema := datasourceSchemaFromResourceSchema(resourceScalewayAccountSSKKey().Schema)
+	addOptionalFieldsToSchema(dsSchema, "name")
+
+	dsSchema["name"].ConflictsWith = []string{"ssh_key_id"}
+	dsSchema["ssh_key_id"] = &schema.Schema{
+		Type:         schema.TypeString,
+		Optional:     true,
+		Computed:     true,
```
Comment (jerome-quere) [inline scaleway/data_source_account_ssh_key.go:21]: Should not be marked as computed

File: scaleway/data_source_account_ssh_key.go [hunk 2]
```diff
@@ -11,35 +11,26 @@ import (
...
+		Description:  "The ID of the SSH key",
+		ValidateFunc: validationUUIDorUUIDWithLocality(),
```
Comment (jerome-quere) [inline scaleway/data_source_account_ssh_key.go:23]: ssh-key are never localized

**Review by jerome-quere — CHANGES_REQUESTED** [review]
view comments

**Comment by remyleone** [comment]
@jerome-quere do you see other resources that should be normalized?

**Review by jerome-quere — APPROVED** [review]

**Pull request merged**